cmd/shipperctl/cmd/apply.go

```diff
@@ -0,0 +1,371 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+
+	"github.com/bookingcom/shipper/cmd/shipperctl/config"
+	"github.com/bookingcom/shipper/cmd/shipperctl/configurator"
+	"github.com/bookingcom/shipper/pkg/crds"
+)
+
+var applyCmd = &cobra.Command{
+	Use:   "apply",
+	Short: "Set up management and application clusters to work with Shipper",
```

Comment:
I appreciate these descriptions, it gives an idea what the package does. I'm also wondering if it makes any sense to write regular package godoc-friendly comments so the automatically built documentation will include it right away?